src/gamefalloutnv.cpp

```diff
@@ -149,9 +149,7 @@ QList<ExecutableInfo> GameFalloutNV::executables() const
   ExecutableInfo launcher("Fallout Launcher", findInGameFolder(getLauncherName()));
   QList<ExecutableInfo> extraExecutables =
       QList<ExecutableInfo>()
-      << ExecutableInfo("Fallout Mod Manager", findInGameFolder("fomm/fomm.exe"))
-      << ExecutableInfo("Construction Kit", findInGameFolder("geck.exe"))
-      << ExecutableInfo("BOSS", findInGameFolder("BOSS/BOSS.exe"))
+      << ExecutableInfo("GECK", findInGameFolder("geck.exe"))
```

Comment:
You can keep the 'GECK' name change I suppose, but I think we'd prefer to keep the other executables for legacy reasons. They will only populate if these applications were already installed by the user so there's no clear reason to remove a convenience layer here.